src/pages/classes/index.tsx

```diff
@@ -61,6 +63,9 @@ export default function Classes() {
 	} = useGetClasses(pagination.pageIndex, pagination.pageSize, semesterTag);
 	const classesColumnDefs = useClassesTableData();
 
+	const searchValue = useStudentsStore((state) => state.searchValue);
+	const { filteredClasses } = useClassesSearch(searchValue, classes);
+
```

Comment:
Rozumiem idee i nawet fajnie to działa, żeby współdzielić stan searcha na stronach, ale nie podoba mi się, że w takim razie to siedzi w `students.store`, zrób store dla filtrów. `filters.store.ts` i tam będziemy współdzielić takie rzeczy.